### ISSUE TITLE
A11y_AzureDataStudioAzureArcExtension_AzureArcController_ConnecttoExistingAzureArcController_DeveloperTool: No name property is provided for the edit box "Kube config file path"

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15717]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AzureArcExtension_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#DevTools;#WCAG4.1.2;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio -Azure Arc Extension
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to "Create new Azure Arc Controller"
4. Tab to "Kube config file path" edit box hit "ctrl + shift + i" and check if it has name provided.

### Actual:
No name property is provided for the edit box "Kube config file path"

### Expected:
Name property to the edit box should be assigned according to it's label which is "Kube config file path"

User Impact:
Screen Reader users will not get the proper information about the edit box

**Comment by Charles-Gagnon** [comment]
The edit box has an aria-label attribute and shows as having a name for me : 

![image](https://user-images.githubusercontent.com/28519865/122269725-60d43880-ce92-11eb-92b4-d47c22338562.png)

![image](https://user-images.githubusercontent.com/28519865/122269860-87926f00-ce92-11eb-883e-a35a958ccc75.png)

Closing as this doesn't seem to be an actual issue, let me know if I'm missing something.

**Comment by Charles-Gagnon** [comment]
Re-opening as I looked at the title and realized that the description was incorrect, this is for the connect to EXISTING controller dialog. Not the "Create New Azure Arc Controller" dialog as indicated there.

**Comment by Shivamd1** [comment]
@Charles-Gagnon  Is there any specific version in which issue is not reproducing?

**Comment by Charles-Gagnon** [comment]
The `Create new Azure Arc Controller` (which is what was indicated in the description) dialog has the name property for the edit box as expected.

**Comment by Shivamd1** [comment]
Closing as issue fixed

**Comment by Shivamd1** [comment]
![image](https://user-images.githubusercontent.com/60880975/150949231-ca394bfe-9324-44cc-b4c4-4e3fc40bae6c.png)

**Comment by prabhudevu** [comment]
GithubTags:#Closed;
